config/Dockerfiles/singlehost-test/verify-rpm.sh

```diff
@@ -35,7 +35,6 @@ set -u
 
 set -xo pipefail
 ansible-playbook -v --inventory=${ANSIBLE_INVENTORY} ${PYTHON_INTERPRETER} \
-	--extra-vars "subjects=${TEST_SUBJECTS}" \
```

Comment:
no need to set a variable that is not used :)

Reply:
hmm, but the script has earlier:

```
if [ -z "${TEST_SUBJECTS}" ]; then
    echo "No test subject defined. Exiting..."
    exit 1
fi
```
So what is the reason to have this there? Same applies for TEST_INVENTORY I guess

Reply:
TEST_SUBJECTS is necessary because is the path for qcow2 image that will be used by the inventory. There is not need to pass this variable to the playbook as it will not use it.

TEST_INVENTORY is hardcoded in Dockerfile as /usr/share/ansible/inventory/standard-inventory-qcow2, so it seems fine, although it shouldn't need to specify it. If you want I can remove it...

Reply:
I would leave it there, as STI defines to pass them no? :)